### PR TITLE
feat: add Windows 11 support

### PR DESCRIPTION
**Pull request by lizheming**

### CODE REVIEW

File: pnpm-lock.yaml [hunk 1]
```diff
@@ -14839,7 +14842,7 @@ packages:
     peerDependencies:
       webpack: ^5.0.0
     dependencies:
-      webpack: 5.75.0(webpack-cli@5.0.1)
+      webpack: 5.75.0(webpack-cli@4.10.0)
```
Comment (Mister-Hope) [inline pnpm-lock.yaml:14845]: I think we should revert this change?

Comment (lizheming) [reply pnpm-lock.yaml:14845]: This change is not caused by my modification. I try to get the historical pnpm-lock.yml to overwrite and then install the dependencies, and this modification will be automatically brought. In addition, I see that there is already a dependency of `webpack 5(webpack-cli@4) in pnpm-lock.
